src/lib/vocab/space/SpaceRepo.ts

```diff
@@ -108,4 +108,11 @@ export class SpaceRepo extends PostgresRepo {
 		if (!data.length) throw Error('no space found');
 		return data;
 	}
+
+	async deleteByHub(hub_id: HubId): Promise<Space[]> {
+		const data = await this.sql<Space[]>`
+			DELETE FROM spaces WHERE hub_id=${hub_id}
+		`;
+		return data;
```

Comment:
The type here is off, I looked at similar and we're returning `number` and `data.count` for this kind of call.

```suggestion
	async deleteByHub(hub_id: HubId): Promise<number> {
		const data = await this.sql<any[]>`
			DELETE FROM spaces WHERE hub_id=${hub_id}
		`;
		return data.count;
```

Reply:
oh yeah, that's a copy pasta error 🙃